ts/Series/Heatmap/HeatmapSeries.ts

```diff
@@ -426,18 +447,137 @@ class HeatmapSeries extends ScatterSeries {
      * @private
      */
     public drawPoints(): void {
+        const
+            heatmap = this,
+            heatmapOptions = heatmap.options,
```

Comment:
To keep this consistent with existing code (and easier for colleagues to read), these two variables should be called `series` and `options`.